StaticAnalyzer/tools/apkid

```diff
@@ -0,0 +1,11 @@
+#!/usr/local/opt/python/bin/python2.7
+
+# -*- coding: utf-8 -*-
+import re
+import sys
+
+from apkid import main
```

Comment:
Hi Ajin,

As you have rightly noted, I added the apkid executable to the StaticAnalyzer/tools directory. The reason for this is that, according to the authors of APKiD, these steps are currently necessary to install it:

git clone https://github.com/rednaga/yara-python
cd yara-python
python setup.py install
pip install apkid

They say that the yara-python clone and compile steps are temporarily necessary until the original mantainers of Yara merge their module into the master branch.

Should we wait until these extra steps are no longer necessary before we add apkid to requirements.txt?

Many thanks!